mono/mini/mini-exceptions.c

```diff
@@ -128,29 +130,49 @@ mono_thread_get_managed_sp (void)
 }
 
 static inline int
-mini_abort_threshold_offset (gpointer threshold, gpointer sp)
+mini_abort_threshold_offset (size_t stack_threshold, size_t stack_size)
 {
-	intptr_t stack_threshold = (intptr_t) threshold;
```

Comment:
Can this get a comment saying that the arguments are sometimes sp, and sometimes stack size ?